contracts/test/RevenueDistributionToken.t.sol

```diff
@@ -2516,26 +2516,6 @@ contract RedeemFailureTests is RDTTestBase {
         assertEq(rdToken.allowance(address(shareOwner), address(notShareOwner)), 0);
     }
 
-    function test_redeem_callerNotOwner_infiniteApprovalForCaller() public {
```

Comment:
We want infinite approval behavior. The ERC20 code needs to be changed so that this passes